src/caip.test.ts

```diff
@@ -0,0 +1,42 @@
+import { toCaipChainId, CaipNamespaces } from './caip';
+
+describe('toCaipChainId', () => {
+  // This function relies on @metamask/utils CAIP helpers. Those are being
+  // tested with a variety of inputs.
+  // Here we mainly focus on our own wrapper around those:
+
+  it('returns a valid CAIP-2 chain ID', () => {
+    const namespace = CaipNamespaces.Eip155;
+    const reference = '1';
+    expect(toCaipChainId(namespace, reference)).toBe(
+      `${namespace}:${reference}`,
+    );
+  });
+
+  it.each([
+    // Too short, must have 3 chars at least
+    '',
+    'xs',
+    // Not matching
+    '!@#$%^&*()',
+    // Too long
+    'namespacetoolong',
+  ])('throws for invalid namespaces: %s', (namespace) => {
+    const reference = '1';
+    // eslint-disable-next-line jest/require-to-throw-message
+    expect(() => toCaipChainId(namespace, reference)).toThrow();
```

Comment:
Is there a reason to not include part of the expected error message?

Reply:
Is being lazy a good reason? :trollface: You're right, I'm gonna fix that right now! :)

Reply:
Fixed!